styles/style.css

```diff
@@ -6,43 +6,51 @@
  * Class: Scotland 2017
  */
 
-html,
-body {
-	font-family: "Source Sans Pro","Helvetica Neue",Arial,sans-serif;
+html, body {
+	font-family: "Source Sans Pro", "Helvetica Neue", Arial, sans-serif;
 }
-
-button {
-	font-family: "Source Sans Pro","Helvetica Neue",Arial,sans-serif;
-}
-
 /*  Navigation */
-.navbar-brand > img {
+
+.navbar-brand>img {
 	max-height: 40px;
 	width: auto;
 }
-
 .navbar-light .navbar-toggler {
 	border: 0;
-	right:  0;
+	right: 0;
 }
-
 .navbar-light .navbar-nav .nav-link {
```

Comment:
I think you could reduce the `font-size` on these, and add a little `margin-right`